lib/utils/favorite_locations.dart

```diff
@@ -1,20 +1,21 @@
 import 'dart:collection';
 import 'package:flutter/material.dart';
-import 'location_data.dart';
+
+import './location_data.dart';
 
 class FavoriteLocations extends ChangeNotifier {
-  final Set<LocationData> _currentLocations = {};
+  final List<LocationData> _currentLocations = [];
```

Comment:
Why change this from a Set? I am just curious. :)

Reply:
I change it to a list because now the home route uses the `PageView.builder` and requires an index. Now it's less efficient when adding and removing favorite locations, but they get constructed in a more efficient way